### PR TITLE
Fix/2022 higgs

### PR DESCRIPTION
**Pull request by mafrahm**
This PR adds missing higgs imports in the 2022 campaigns and fixes the corresponding datasets.

Also, the unittests for checking correctness of DAS info have been fixed.

### CODE REVIEW

**Comment by mafrahm** [comment]
Update: I also fixed the remaining disagreements between DAS and cmsdb. Therefore, this PR closes #45

File: cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py [hunk 1]
```diff
@@ -152,20 +152,20 @@
...
+#     info=dict(
+#         nominal=DatasetInfo(
+#             keys=[
+#                 "/GluGluHto2Zto4Nu_PT-150_M-125_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM",  # noqa
+#             ],
+#             n_files=90,
+#             n_events=3734036,
+#         ),
+#     ),
+# )
```
Comment (nprouvost) [inline cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py:168]: What is the reason for commenting this dataset? It is marked as valid in DAS...

File: cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py [hunk 2]
```diff
@@ -944,20 +944,6 @@
...
-    info=dict(
-        nominal=DatasetInfo(
-            keys=[
-                "/WminusH_Hto2Zto4L_M-125p5_TuneCP5_13p6TeV_powheg2-minlo-HWJ-JHUGenV752-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM",  # noqa
-            ],
-            n_files=37,
-            n_events=1730541,
-        ),
-    ),
-)
```
Comment (nprouvost) [inline cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py:960]: What is the reason for deleting this dataset? It is marked as valid in DAS...

File: cmsdb/campaigns/run3_2022_preEE_nano_v12/higgs.py [hunk 3]
```diff
@@ -20,41 +20,41 @@
...
+#     info=dict(
+#         nominal=DatasetInfo(
+#             keys=[
+#                 "/GluGluHtoCC_PT-200_M-125_TuneCP5_13p6TeV_powheg-minlo-pythia8/Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5-v2/NANOAODSIM",  # noqa
+#             ],
+#             n_files=20,
+#             n_events=122144,
+#         ),
+#     ),
+# )
```
Comment (nprouvost) [inline cmsdb/campaigns/run3_2022_preEE_nano_v12/higgs.py:57]: What is the reason for commenting these datasets? They are marked as valid in DAS...

**Review by nprouvost — APPROVED** [review]
Don't understand why some datasets are commented out/deleted, else LGTM

File: cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py (hunk 1 above)
Comment (mafrahm) [reply cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py:168]: yes, but there is no corresponding process in the cmsdb

File: cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py (hunk 2 above)
Comment (mafrahm) [reply cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py:960]: this is generated with m_H=125.5GeV. There are lots of samples with mass variations and we do not consider any of them

File: cmsdb/campaigns/run3_2022_preEE_nano_v12/higgs.py (hunk 3 above)
Comment (mafrahm) [reply cmsdb/campaigns/run3_2022_preEE_nano_v12/higgs.py:57]: yes, but there is no corresponding process in the cmsdb

**Pull request merged**